.github/workflows/review.yml

```diff
@@ -61,7 +61,7 @@ jobs:
           K8S_SECRET_MAIL_MAILGUN_KEY: ${{ secrets.GH_STAGING_MAILGUN_API_KEY }}
           K8S_SECRET_MAIL_MAILGUN_DOMAIN: "hel.fi"
           K8S_SECRET_MAIL_MAILGUN_API: "https://api.eu.mailgun.net/v3"
-          K8S_SECRET_HKM_PBW_API_ENDPOINT: "https://payform.bambora.com/pbwapi"
+          K8S_SECRET_HKM_PBW_API_ENDPOINT: ${{ secrets.GH_QA_PBW_API_ENDPOINT }}
```

Comment:
Should the key be `GH_QA_HKM_PBW_API_ENDPOINT`?

Reply:
The key is in line with the rest of the `PBW` keys
`GH_STABLE_PBW_API_KEY` and `GH_STABLE_PBW_SECRET_KEY`

Reply:
Hmm... Vault currently has them with the HKM_ prefix:

![image](https://user-images.githubusercontent.com/54528563/118463915-f69c6c80-b708-11eb-8136-05bdd2c9db9b.png)

That's why I was thinking that they should be named in a similar manner.

Reply:
Now that I think of it, those names in the Vault might be wrong because they are corresponding to the names we map in the `settings.py`

Reply:
Ah, OK. In that case I think this PR is fine then. We'll just have to fix the Vault key names.